python/ql/src/meta/analysis-quality/CallGraph.ql

```diff
@@ -0,0 +1,16 @@
+/**
+ * @name Call graph
+ * @description An edge in the points-to call graph.
+ * @kind problem
+ * @problem.severity recommendation
+ * @id py/meta/points-to-call-graph
```

Comment:
I don't see any points-to specific implementation details, so I think we can remove that part from the query-id?

```suggestion
 * @id py/meta/call-graph
```

Reply:
oh, and if we do this, we should probably do it in the description as well

Reply:
`DataFlowCallableValue` is a `DataFlowCallable` explicitly backed by a `CallableValue`. This is why the query is points-to specific. 

I wanted to keep the query-id separate, so as to avoid potential confusion if we ever want to compare results between different meta-queries in the future.

I considered using the `RelevantCall` machinery, but for now I just wanted to get an impression of the complete picture. We can always add it later if needed.

Reply:
I misread `DataFlowCallableValue`, so never mind me then.